tests/basic/src/main/java/mil/emp3/dev_test_sdk/MainActivity.java

```diff
@@ -538,9 +534,9 @@ protected IGeoPosition getRandomCoordinate() {
         return oPosList;
     }
 
-    private java.util.List<IGeoPosition> getPositions(int iCount, double dAltitude) {
+    private List<IGeoPosition> getPositions(int iCount, double dAltitude) {
```

Comment:
In general, I enforce final variables everywhere it is possible.  I know this is outside the scope of this PR, but we should start transitioning that way for the EMP codebase.